src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+computed distances(based on a distance metric) in the transformed space of rowᵢ in
+`Xtest` to the centroid of each class.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `<:OrderedFactor(2)` or `<:Multiclass(2)`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `normalize=true`: Option to normalize the between class variance for the number of
+   observations in each class, one of `true` or `false`.
+- `out_dim`: The dimension of the transformed space to be used by `predict` and
+   `transform` methods, automatically set if 0 is given (default).
+- `dist=SqEuclidean`: The distance metric to use when performing classification
+   (to compare the distance between a new point and centroids in the transformed space),
+   an alternative choice can be the `CosineDist`.
+
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+BUG: RENAME, Make note on top of PR for inconsistency
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `explained_variance_ratio`: The ratio of explained variance to total variance. Each dimension corresponds to an eigenvalue.
+- `classes`: The classes seen during model fitting.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `mean`: The mean vector.
+- `class_weights`: The weights of each class.
+- `nc`: The number of classes.
+
+# Examples
+
+```
+using MLJ
+
+sLDA = @load SubspaceLDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = sLDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, X)
+labels = predict_mode(mach, X)
+```
+
+See also
+[`LDA`](@ref), [`BayesianLDA`](@ref), [`BayesianSubspaceLDA`](@ref)
+"""
+SubspaceLDA
+"""
+$(MMI.doc_header(BayesianSubspaceLDA))
+
+
+`SubspaceLDA`: Bayesian Multiclass linear discriminant analysis. Suitable for high dimensional data
```

Comment:
Something wrong here. We just did subspace LDA above. This should be FactorAnalysis